### ISSUE TITLE
 My Train_loss  is always 2.0~3.0

### ISSUE THREAD

**Issue by KaiiZhang** [EternityZY/FCN-TensorFlow#3]
I have set the learn rate to 5e-7 and MAX_ITERATION to 100001, but it still does not work
What should i do?
Thanks!

**Comment by yeLer** [comment]
Have you solved the problem? I miss the the state as you,my training and validation loss are always between 3 to 4. The visual results from different images are same.
![2019-05-16_200908](https://user-images.githubusercontent.com/16631986/57852731-86576980-7816-11e9-942d-6969b1717844.png)
![2019-05-16_200921](https://user-images.githubusercontent.com/16631986/57852732-88212d00-7816-11e9-8b64-68f589d24d50.png)
